pkg/manifests/manifests.go

```diff
@@ -70,7 +70,13 @@ const (
 )
 
 const (
-	containerNameRTE = "resource-topology-exporter"
+	containerNameRTE                = "resource-topology-exporter"
+	rteNotifierVolumeName           = "host-run-rte"
+	rteSysVolumeName                = "host-sys"
+	rtePodresourcesSocketVolumeName = "host-podresources-socket"
+	rteKubeletDirVolumeName         = "host-var-lib-kubelet"
+	rteNotifierFileName             = "notify"
+	hostNotifierDir                 = "/run/rte"
```

Comment:
this should be overridable with a parameter as well (in the future)

Reply:
ye, see my comment above, I just unsure regarding to the SELinux.